introduction/theTechnology.md

```diff
@@ -0,0 +1,59 @@
+
+<a name="jsCoBrowsing"></a>
+##### Javascript Solutions{#jsCoBrowsing}
+
+With Javascript based solutions, a widget is created in which the Javascript of the original page is being loaded. This is a time consuming process since requests from the user in control continuously need to be sent to the website, then to the co-browsing solution, then back to the controlling user as well as the followers.
+
+Not only is this an exhausting process, there are many limitations to it. For example, since the iframes on the original page come from an external Javascript source, they can't be loaded within the Javascript that the co-browsing solution sends over to the users. Also, it is unsafe to handle logged-in sessions, as they are usually only possible if login credentials are sent to the followers; otherwise the followers do not have access to session-specific data.
+
+<a name="surflyCoBrowsing"></a>
+##### Co-browsing with Surfly{#surflyCoBrowsing}
+
+Surfly distinguishes itself by using a unique combination of Javascript and a smart content rewriting proxy. This is how it works:
+
+* when the leader starts a session, the browser sends a request to the Surfly proxy
+* the request is then modified in a way that it looks as if the original request came from surfly.com
+* this request is then sent to the original site that the leader wants to co-browse
+* the website sends back the request to the Surfly proxy
+* the Surfly proxy then modifies the data so that it can be loaded into an iframe that rests on top the original page
+* both the leader and the follower now interact with the website as it is loaded within the iframe. From this point, there's only communication between the user's browsers and the proxy, and the requests don't continuously have to be sent to the original website anymore
+
+![surfly-scheme](../images/surfly-scheme.png)
+
+This approach enables us to overcome cross-domain policies and have all elements on the site (including iframes) function correctly within the co-browsing session. In addition, all visual updates can be efficiently captured. The proxy approach also allows us to provide both the user and the agent with a very smooth co-browsing experience. This means that:
+
+* Surfly is safe; all connections to Surfly are encrypted and security tokens or passwords will never be sent to the follower. The secure proxy allows ‘logged-in sessions’. This means that secure sessions with session-specific resources just work. Next, we automatically make sure that all content will work correctly with the co-browsing session: even those that are not served from the same domain, such as widgets or plugins.
+* With Surfly there is no need to install any plugin or software; it just works within the browser tab.
+* We do not share the state between the two different browsers, but we only send over the visual changes of one to the other. This makes our approach much more secure and robust.
```

Comment:
Also, faster, since the information about the content changes is way smaller than a stream of images